drivers/disk/usdhc.c

```diff
@@ -2634,35 +2634,41 @@ static int usdhc_board_access_init(struct usdhc_priv *priv)
 	}
 
 	if (!priv->detect_gpio) {
-		LOG_INF("USDHC detection other than GPIO not implemented!");
-		return 0;
-	}
+		LOG_INF("USDHC detection other than GPIO");
```

Comment:
Please update the commit message `drivers: disk: Update NXP USDHC driver` to indicate that this update is to support the IP version on RT600

Reply:
This is available to USDHC IP version present on RT1064 and other RT platform. This pin was not used in those SoC's, instead the detect signal was connected to a GPIO pin.